src/html/templates/toc.hbs

```diff
@@ -0,0 +1,34 @@
+<div class="toc">
+  <div>
+    {{~#if usages~}}
+      {{~> usages usages ~}}
+    {{~/if~}}
+
+    {{~#if top_symbols~}}
+      <nav class="topSymbols">
+        <h3>Symbols</h3>
+        <ul>
+          {{~#each top_symbols.symbols~}}
+            <li>
+              <a href="{{this.href}}" title="{{this.name}}">
+                {{~> doc_node_kind_icon this.kind~}}
+                <span class="hover:bg-{{this.kind.[0].kind}}/15">{{this.name}}</span>
+              </a>
+            </li>
+          {{~/each~}}
+        </ul>
+        <a class="flex items-center gap-0.5" href="{{top_symbols.all_symbols_href}}">
+          <span class="leading-none">view all {{top_symbols.total_symbols}} symbols</span>
+          {{~> icons/arrow ~}}
+        </a>
```

Comment:
Only show if more than 5 - otherwise you'll have to deal with "see all 1 symbols" (wrong plural symbols)